src/linter/html/HtmlReporter.ts

```diff
@@ -44,10 +45,19 @@ export default class HtmlReporter {
 		} else {
 			startPos = node.start;
 		}
-		this.#context.addLintingMessage(this.#resourcePath, id, args, {
+		const position = {
 			line: startPos.line + 1,
 			column: startPos.character + 1,
-		});
+		};
+
+		const message = {
+			id,
+			args,
+			position,
+			fix,
+		};
+
+		this.#context.addLintingMessages(this.#resourcePath, [message]);
```

Comment:
Is there a reason to switch from `this.#context.addLintingMessage` to `this.#context.addLintingMessages`?

Reply:
Done